latex/Research_Strategy.tex

```diff
@@ -277,83 +277,71 @@
+Our goal is to assess the current state of physical calculations of these properties, and drive innovation. 
+Distribution coefficients serve well for this, as they relate to transfer free energies from aqueous high-dielectric to low-dielectric environments, capturing many of the characteristics of transfer of drugs from water into binding sites but absent challenges with receptor conformational sampling and specific ligand-receptor interactions.
+In SAMPL5, distribution coefficients were challenging enough that many methods performed poorly, with even the best methods having accuracies less than would have been expected based on hydration free energies in water~\cite{bannan_blind_2016}, yet failures were informative and the major sources of error were issues which will also plague prediction of ligand-receptor interactions.
 In some respects, distribution coefficients posed the ideal SAMPL challenge, hitting the sweet spot in terms of difficulty---difficult enough that clear failures were frequent and there is much room for improvement, but not so difficult that the reasons for failure were unclear in general. 
-Still, the challenge could have been improved by follow-up experiments to re-check some of the experimental results~\cite{paranahewage_predicting_2016, klamt_prediction_2016, bannan_blind_2016, rustenburg_measuring_2016}---but without funding for someone to continue working in this space, this has so far been impossible.
-{\color{red}[JDC: Turn this into a positive statement: This exercise the utility that funded, targeted follow-up experiments could have in addressing issues of experimental discrepancies shared by \emph{all} models.]}
+Still, many models disagreed with experiment in consistent ways for particular compounds~\cite{paranahewage_predicting_2016, klamt_prediction_2016, bannan_blind_2016, rustenburg_measuring_2016}, revealing the impact that targeted follow-up experiments (such as those we will conduct here) could have on improving  models.
+Our choice of cyclohexane-water distribution coefficients rather than octanol-water was motivated by both complexities with octanol experiments (~\cite{bannan_calculating_2016}, making it difficult to ensure the system modeled is the same one simulated) and potential sampling problems in octanol~\cite{Kollman:1996:AccountsofChemicalResearch}, though challenges we propose here will broaden past cyclohexane.
+
+%Talk about limitations of approach -- lack of ability to do follow up expeirments, get the dynamic range we wanted, test things again after SAMPL? 
+% Sustained funding will allow us to do that. 
+% (This will also help answer the argument that we don't need funding for this...)
+\textbf{This new experimental data is critical to make SAMPL what it ought to be. }
```

Comment:
Maybe something like "A funded, coordinated effort allows the targeted collection of specific datasets designed to focus on the problems of highest relevance. Additionally, with multiple investigators and collaborators, we are poised to respond and adapt to new challenges and opportunities that emerge in a manner that was not possible in the previous incarnation of SAMPL".